app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    Rails.logger.debug(params.inspect)
```

Comment:
**[OPTIONAL]**

- [ ] It is always good practice to remove debugging code once you are done. This improves your app security and ensure sensitive information of your application is not exposed